src/main/java/nl/tudelft/pixelperfect/event/EventParameter.java

```diff
@@ -0,0 +1,72 @@
+package nl.tudelft.pixelperfect.event;
+
+/**
+ * Class representing a single parameter of an Event, having either a discrete value
+ * (EventParameterValue) or a continuous one (Integer).
+ * 
+ * @author Jesse Tilro
+ *
+ */
+public class EventParameter {
+
+  private String key;
+  private EventParameterValues value;
+  private int number;
```

Comment:
I agree with Floris, it's not a huge deal right now, but try to always give it a descriptor and an identifier.  Perhaps:  "parameterNumber" or "thisValCountsInts."